spec/controllers/fines_controller_spec.rb

```diff
@@ -3,18 +3,18 @@
 require 'rails_helper'
 
 RSpec.describe FinesController do
-  let(:mock_patron) { instance_double(Symphony::Patron, group?: false, barcode: '1234') }
+  let(:mock_patron) { instance_double(Folio::Patron, group?: false, barcode: '1234') }
 
   let(:fines) do
     [
-      instance_double(Symphony::Fine, key: '1', sequence: '1', owed: '12', status: 'BADCHECK')
+      instance_double(Folio::Account, key: '1', sequence: '1', owed: '12', status: 'BADCHECK')
```

Comment:
might be another thing that doesn't need to go in this PR, but i believe 'sequence' is a symphony-specific thing because we needed to submit this weird "billing sequence" param to the ILS, which we no longer do.